introduction.md

```diff
@@ -1 +1,4 @@
-Hello. This is the introduction file. My favourite codeschool path is Javascript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?